lib/srv/sshserver.go

```diff
@@ -526,6 +537,9 @@ func (s *Server) HandleNewChan(nc net.Conn, sconn *ssh.ServerConn, nch ssh.NewCh
 	}
 
 	switch channelType {
+	case "terminal-size-notifier":
```

Comment:
let's use constant here and may be prefix with `x-teleport` to show that it's our extension